plaso/parsers/czip_plugins/interface.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+"""czip_interface contains basic interface for compound ZIP plugins within
+Plaso.
+
+CompoundZIPPlugin defines the attributes necessary for registration, discovery
+and operation of plugins for czip files which will be used by CompoundZIPParser.
+"""
+
+from __future__ import unicode_literals
+
+from plaso.lib import errors
+from plaso.parsers import logger
+from plaso.parsers import plugins
+
+class CompoundZIPPlugin(plugins.BasePlugin):
+  """This is and "abstract" class from which plugins should be based."""
+
+  # REQUIRED_ITEMS is a list of items required by a plugin.
+  # This is used to understand whether a plugin is suited for a given compound
+  # ZIP archive.
+  # This is expected to be overridden by actual plugins.
+  # Ex. frozenset(['[Content_Types].xml', '_rels/.rels', 'docProps/core.xml'])
+  REQUIRED_ITEMS = frozenset(['any'])
```

Comment:
Rename this to REQUIRED_PATHS? Since these are file paths inside the archive.

What does 'any' mean here? Just leave it empty if there's no content left.

Reply:
My fear was that an empty set is a superset for everything, so forgetting to override that field with a proper value would result in a sneaky bug. Let me know which approach do you prefer.